src/dymaptic.GeoBlazor.Core/Model/Locator.cs

```diff
@@ -0,0 +1,89 @@
+﻿using dymaptic.GeoBlazor.Core.Components.Geometries;
+using dymaptic.GeoBlazor.Core.Objects;
+using Microsoft.JSInterop;
+
+namespace dymaptic.GeoBlazor.Core.Model;
+
+/// <summary>
+///    Represents a geocode service resource exposed by the ArcGIS Server REST API. It is used to generate candidates for an address. It is also used to generate batch results for a set of addresses.
+///    Set the URL to the ArcGIS Server REST resource that represents a Locator service, for example: https://geocode.arcgis.com/arcgis/rest/services/World/GeocodeServer.
```

Comment:
Per the discussion.  The doing batch item geocoding will need the user to have an api key setup to do those geocoding calls. reference to the developer.arcgis.com and setting up the api key to have the permissions set to "Geocode (Stored)" vs the default setting... Geocode (Not Stored)

![image](https://github.com/dymaptic/GeoBlazor/assets/15984068/684a99b6-86c8-49a5-b425-652e56bec26c)

Reply:
added comments to the locator class